src/ripple/net/SSLHTTPDownloader.h

```diff
@@ -22,6 +22,7 @@
 
 #include <ripple/basics/Log.h>
 #include <ripple/core/Config.h>
+#include <ripple/net/HTTPClientSSLContext.h>
```

Comment:
`RegisterSSLCerts.h` no longer needs to be included from the source file.

Reply:
done